Src/AutoFixture/OmitOnRecursionBehavior.cs

```diff
@@ -26,8 +26,12 @@ public OmitOnRecursionBehavior() : this(DefaultRecursionDepth)
         /// Initializes new instance of the <see cref="OmitOnRecursionBehavior" /> class with specific recursion depth.
         /// </summary>
         /// <param name="recursionDepth">The recursion depth at which the request will be omitted.</param>
+        /// <exception cref="System.ArgumentOutOfRangeException">recursionDepth is less than zero.</exception>
         public OmitOnRecursionBehavior(int recursionDepth)
         {
+            if (recursionDepth < 1)
+                throw new ArgumentOutOfRangeException("recursionDepth");

```

Comment:
Would it be more helpful to the user if the exception message explains what is the acceptable range?